src/commands/syncConfig.js

```diff
@@ -0,0 +1,20 @@
+import chalk from 'chalk';
+import normalizeConfig from '../helpers/normalizeConfig';
+import forEachProject from '../traversal/forEachProject';
+
+import runSyncModules from '../helpers/runSyncModules';
+
+export { default as builder } from '../helpers/validateCommand';
+export const desc = 'Sync variation consumers.';
+export const command = 'sync';
+
+export const handler = (config) => {
+  const { sync, projectNames, projects } = normalizeConfig(config);
+
+  forEachProject(projects, projectNames, (projectName, projectConfig) => {
```

Comment:
we'll want to require the `requires` here, so all the right registrations are set up